Dockerfile

```
@@ -7,29 +7,33 @@ COPY ./web .
-FROM golang:alpine AS builder2
+FROM swr.cn-north-4.myhuaweicloud.com/ddn-k8s/docker.io/golang:alpine3.21 AS builder2
 
 ENV GO111MODULE=on \
     CGO_ENABLED=0 \
-    GOOS=linux
+    GOOS=linux \
+    GOPROXY=https://goproxy.cn
 
 WORKDIR /build
 
 ADD go.mod go.sum ./
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)